### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by tdviet**
# Summary 

Adding information for accessing GPU sites

**Related issue :**

### CODE REVIEW

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 1]
```diff
@@ -6,19 +6,32 @@ description: >
...
 | ---- | ------------------------ | ------- | ----------------------- | ----------------- |
 | IISAS-GPUCloud | up to 2 NVIDIA Tesla K20m | gpu1cpu6, gpu2cpu12 | acc-comp.egi.eu, eosc-synergy.eu, enmr.eu, training.egi.eu | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
 | IISAS-Nebula | up to 2 NVIDIA Tesla K20m | resource_tpl#large_gpu | acc-comp.egi.eu, biomed | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
-| IFCA-LCG2 | up to 4 NVIDIA GT200GL | | | Pay-per-use |
-| IFCA-LCG2 | up to 1 NVIDIA TITAN X | | | Pay-per-use |
-| IFCA-LCG2 | up to 10 NVIDIA GeForce GTX 1080Ti | | | Pay-per-use |
+| IFCA-LCG2 | up to 4 NVIDIA GT200GL, up to 1 NVIDIA TITAN X, up to 10 NVIDIA GeForce GTX 1080Ti | | | Pay-per-use |
 | CESNET-MCC | up to 2 NVIDIA T4 | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com |Sponsored, conditions to be negotiated |
+
+# Access to GPU resources on EGI Cloud:
```
Comment (gwarf) [inline content/en/users/cloud-compute/gpgpu/_index.md:25]:
```suggestion
# Access to GPU resources on EGI Cloud
```

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 2]
```diff
@@ -6,19 +6,32 @@ description: >
...
 | IISAS-Nebula | up to 2 NVIDIA Tesla K20m | resource_tpl#large_gpu | acc-comp.egi.eu, biomed | Sponsored access for limited testing, conditions to be negotiated for long-term usage |
-| IFCA-LCG2 | up to 4 NVIDIA GT200GL | | | Pay-per-use |
-| IFCA-LCG2 | up to 1 NVIDIA TITAN X | | | Pay-per-use |
-| IFCA-LCG2 | up to 10 NVIDIA GeForce GTX 1080Ti | | | Pay-per-use |
+| IFCA-LCG2 | up to 4 NVIDIA GT200GL, up to 1 NVIDIA TITAN X, up to 10 NVIDIA GeForce GTX 1080Ti | | | Pay-per-use |
 | CESNET-MCC | up to 2 NVIDIA T4 | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com |Sponsored, conditions to be negotiated |
+
+# Access to GPU resources on EGI Cloud
+
+GPUs sites can be accessed by different ways: via site-specific dashboards and endpoints or via common federated-cloud services like common Horizon dashboard, VMOps dashboard, Infrastructure manager.
```
Comment (gwarf) [inline content/en/users/cloud-compute/gpgpu/_index.md:27]:
```suggestion
GPUs sites can be accessed by different ways: via site-specific dashboards
and endpoints or via common federated-cloud services like common Horizon
dashboard, VMOps dashboard, Infrastructure manager.
```

**Comment by gwarf** [comment]
Thanks Viet!
I've pushed a commit for fixing a markdown issue and for updating the style to follow our rules.

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 3]
```diff
@@ -6,19 +6,35 @@ description: >
...
+| CESNET-MCC     | up to 2 NVIDIA T4                                                                  | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com       | Sponsored, conditions to be negotiated                                                |
+
+# Access to GPU resources on EGI Cloud
+
+GPUs sites can be accessed by different ways: via site-specific dashboards and
+endpoints or via common federated-cloud services like common Horizon dashboard,
+VMOps dashboard, Infrastructure manager.
+
+Site-specific dashboards and endpoints are described in the folling table:
```
Comment (enolfc) [inline content/en/users/cloud-compute/gpgpu/_index.md:30]:
```suggestion
Site-specific dashboards and endpoints are described in the following table:
```

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 4]
```diff
@@ -6,19 +6,35 @@ description: >
...
+| IFCA-LCG2      | up to 4 NVIDIA GT200GL, up to 1 NVIDIA TITAN X, up to 10 NVIDIA GeForce GTX 1080Ti |                                                                                                                                                                                            |                                                            | Pay-per-use                                                                           |
+| CESNET-MCC     | up to 2 NVIDIA T4                                                                  | hpc.8core-64ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-1080-glados, hpc.38core-372ram-nvidia-1080-glados, hpc.19core-176ram-nvidia-2080-glados, hpc.38core-372ram-nvidia-2080-glados | vo.clarin.eu, biomed, eosc-synergy.eu, peachnote.com       | Sponsored, conditions to be negotiated                                                |
+
+# Access to GPU resources on EGI Cloud
+
+GPUs sites can be accessed by different ways: via site-specific dashboards and
+endpoints or via common federated-cloud services like common Horizon dashboard,
+VMOps dashboard, Infrastructure manager.
```
Comment (enolfc) [inline content/en/users/cloud-compute/gpgpu/_index.md:28]:
```suggestion
VMOps dashboard, or Infrastructure manager.
```

**Review by enolfc — COMMENTED** [review]
Thans @tdviet! LGTM. We probably need a better place for listing the dashboards as this a generic info. Fixing 2 small typos and then merging.

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**